components/Faq.js

```diff
@@ -1,7 +1,130 @@
 import React from "react";
+import Accordion from "react-bootstrap/Accordion";
+
+let colorCount = 1;
+
+const infoArray = [
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+	{
+		question: `quam quisque id diam vel quam elementum 
+		pulvinar etiam non quam lacus suspendisse faucibus interdum`,
+		answer: `Lorem ipsum dolor sit amet, consectetur 
+		adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ullamcorper sit 
+		amet risus nullam eget felis eget nunc lobortis. 
+		Ultrices dui sapien eget mi. Etiam tempor orci eu lobortis 
+		elementum. Tincidunt nunc pulvinar sapien et ligula ullamcorper.`,
+	},
+]; // dummy data
+
+const colors = [
```

Comment:
Which colors are you using? There is a copy here and in `tailwind.config.js`, place them in only one location

Reply:
Updated in latest commit!